<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AboutWidget</class>
 <widget class="QWidget" name="AboutWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>434</width>
    <height>284</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>About</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../icons/info_black.svg</normaloff>../icons/info_black.svg</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QLabel" name="logo">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>100</width>
     <height>100</height>
    </rect>
   </property>
   <property name="pixmap">
    <pixmap>../icons/logo.svg</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="title_label">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>30</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Shuffle</string>
   </property>
  </widget>
  <widget class="QLabel" name="copyright_label">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>90</y>
     <width>211</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Copyright (c) 2022 Renat Skosarev</string>
   </property>
  </widget>
  <widget class="QLabel" name="description_label">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>70</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Audio Player</string>
   </property>
  </widget>
  <widget class="QLabel" name="version_title_label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>150</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Version:</string>
   </property>
  </widget>
  <widget class="QLabel" name="lib_label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>190</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Libraries:</string>
   </property>
  </widget>
  <widget class="QLabel" name="pyqt5_label">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>190</y>
     <width>55</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>• PyQt5</string>
   </property>
  </widget>
  <widget class="QLabel" name="pillow_label">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>210</y>
     <width>55</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>• Pillow</string>
   </property>
  </widget>
  <widget class="QLabel" name="tinytag_label">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>230</y>
     <width>55</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>• tinytag</string>
   </property>
  </widget>
  <widget class="QPushButton" name="github_button">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>210</y>
     <width>181</width>
     <height>28</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
background-color: #6240C2;
border-radius: 10px;
color: rgb(255, 255, 255);
}

QPushButton:hover{
   background-color:#4422A4;
}</string>
   </property>
   <property name="text">
    <string>GitHub</string>
   </property>
  </widget>
  <widget class="QLabel" name="version_label">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>150</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="developer_label">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>150</y>
     <width>171</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Developed by Renat Skosarev</string>
   </property>
  </widget>
  <widget class="QLabel" name="designer_label">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>170</y>
     <width>171</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Designed by Eldar Sayranov</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
